utility/move-template-to-project.js

```diff
@@ -0,0 +1,48 @@
+const path = require('path');
+const extname = require('path').extname;
+const stringUtils = require('./string');
+const fs = require('fs');
+const Mustache = require('mustache');
+
+const applyTemplateToFile = (filePath, templateOptions) => {
+    const tags = ['<%=', '%>'];
+    const fileContent = fs.readFileSync(filePath);
+    const content = Mustache.render(fileContent.toString(), templateOptions, {}, tags);
+
+    return content;
+};
+
+const addPageToApp = (pageName, pathToPage, templatePagesPath) => {
+    fs.readdirSync(templatePagesPath).forEach((pageItem) => {
+        const templateOption = { pageName, title: stringUtils.humanize(pageName) };
+        const content = applyTemplateToFile(path.join(templatePagesPath, pageItem), templateOption);
+        fs.writeFileSync(path.join(pathToPage, `${pageName}${extname(pageItem)}`), content);
+    });
+};
+
+const moveTemplateFilesToProject = (templateFolder, appPath, templateOptions, pathToFileRelativelyRoot) => {
+    const relativePath = pathToFileRelativelyRoot || '';
+    const pathToFiles = path.join(templateFolder, relativePath);
+
+    fs.readdirSync(pathToFiles).forEach(file => {
```

Comment:
I suppose this method could be faster if replace `forEach` with parallel asynchronous iterations.